libr/crypto/p/crypto_base91.c

```diff
@@ -0,0 +1,41 @@
+#include <r_lib.h>
+#include <r_crypto.h>
+#include <r_util.h>
+
+#define INSIZE 32768
+
+static bool base91_use(const char *algo) {
+	return !strcmp (algo, "base91");
+}
+
+static int update(RCrypto *cry, const ut8 *buf, int len, bool to_encode) {
+	int olen = INSIZE; //a way to optimise memory allocation.
```

Comment:
For encoding, since base91 has better compression than base64, I can get upper limit on size of encoded buffer (same as length for base64 encoding), but for decoding, I am not able to figure out a way to reduce such large memory allocation. Please suggest any idea for this.